server/app.py

```diff
@@ -176,4 +176,14 @@
 app.error_handler.add(NotFound, missing_handler)
 
 if __name__ == '__main__':
-	app.run(host='0.0.0.0', port=8080)
\ No newline at end of file
+	app.run(host='0.0.0.0', port=8080)
+
+class Dog:

```

Comment:
Please put this class in it's own file. Lets do it like this. Create an `entity` directory and make a dog.py file.